commit aabbc8bbf3ed90878c29186c31f76156c3797629 (refs/changes/09/4209/2)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2018-06-13T22:19:24+00:00 (1 year, 4 months ago)
    
    MANTA-3761 MorayCleanerStream asserts option that isn't passed in moray_gc.js

diff --git a/bin/moray_gc.js b/bin/moray_gc.js
index 98487f6..87094f3 100755
--- a/bin/moray_gc.js
+++ b/bin/moray_gc.js
@@ -46,6 +46,7 @@ var MORAY_CLEANUP_PATH = '/' + MANTA_USER + '/stor/manta_gc/moray';
 var PID_FILE = '/var/tmp/moray_gc.pid';
 var CRON_START = new Date();
 var MORAY_CLEANER = lib.createMorayCleaner({ log: LOG, batchSize: 1000 });
+var MANTA_DELETE_BUCKET = 'manta_delete_log';
 MORAY_CLEANER.on('error', function (err) {
         LOG.fatal(err);
         var returnCode = auditCron(err);
@@ -80,7 +81,8 @@ function cleanShardOneObject(log, shard, input, cb) {
                  */
                 var mcs = MORAY_CLEANER.cleanStream({
                         shard: shard,
-                        object: input
+                        object: input,
+                        bucket: MANTA_DELETE_BUCKET
                 });
 
                 mcs.once('error', function (mcsErr) {
